src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -0,0 +1,8 @@
+cfg_if! {
+    if #[cfg(target_arch = "x86_64")] {
```

Comment:
You can feel free to inline everything in this file if there's only one definition, but either way works fo rme